common/lib/xmodule/xmodule/css/sequence/display.scss

```diff
@@ -57,279 +60,175 @@ nav.sequence-nav {
-          //   font-size: 12px;
-          //   color: #aaa;
-          // }
-
-          &:hover, &:focus {
-            background-color: $white;
-            background-repeat: no-repeat;
-            background-position: center 14px;
-          }
-        }
-
-        //video
-        &.seq_video {
-          &.inactive {
-            background-image: url('../images/sequence-nav/video-icon-normal.png');
-          }
-
-          &.visited {
-            background-image: url('../images/sequence-nav/video-icon-visited.png');
-          }
-
-          &.active {
-            @extend .active;
-            background-image: url('../images/sequence-nav/video-icon-current.png');
-          }
-        }
-
-        //other
-        &.seq_other {
-          &.inactive {
-            background-image: url('../images/sequence-nav/document-icon-normal.png');
-          }
+        @extend .block-link;
+      }
 
-          &.visited {
-            background-image: url('../images/sequence-nav/document-icon-visited.png');
-          }
+      ol {
+        position: absolute;
 
-          &.active {
-            background-image: url('../images/sequence-nav/document-icon-current.png');
-          }
-        }
+        li {
+          display: table-cell;
+          min-width: 20px;
 
-        //vertical & problems
-        &.seq_vertical, &.seq_problem {
-          &.inactive {
-            background-image: url('../images/sequence-nav/list-icon-normal.png');
-          }
+          a {
+            @extend %ui-fake-link;
+            @include transition(none);
+            width: ($baseline*5);
+            height: ($baseline*2);
+            margin: 0;
+            background-position: center 14px;
+            background-repeat: no-repeat;
+            border: 1px solid transparent;
+            display: block;
+            padding: 0;
+            position: relative;
 
-          &.visited {
-            background-image: url('../images/sequence-nav/list-icon-visited.png');
-          }
+            &:hover, &:focus {
+              background-color: $white;
+              background-repeat: no-repeat;
+              background-position: center 14px;
+            }
 
-          &.active {
-            background-image: url('../images/sequence-nav/list-icon-current.png');
-          }
+            &.active {
+              @extend %ui-depth1;
+              background-color: $white;
 
-          &.progress-none {
-            background-image: url('../images/sequence-nav/list-unstarted.png');
-          }
+              &:hover, &:focus {
+                background-color: $white;
+                background-repeat: no-repeat;
+                background-position: center 14px;
+              }
+            }
 
-          &.progress-some, &.progress-in_progress {
-            background-image: url('../images/sequence-nav/list-unfinished.png');
-          }
+            //video
+            &.seq_video {
+              &.inactive {
```

Comment:
NIT: I know you're just moving style rule order here in your diff, but any place we can change to use the SMACSS stateful classes (like here), I'm all for. This may not be the time (due to more logical/test dependencies), but if its opportunistic, go for it.